nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/README.md

```diff
@@ -1,11 +1,31 @@
-# {{ cookiecutter.pipeline_name }} Documentation
+# {{ cookiecutter.pipeline_name }}
+
+{{ cookiecutter.pipeline_short_description }}
+
+[![Build Status](https://travis-ci.org/{{ cookiecutter.github_repo }}.svg?branch=master)](https://travis-ci.org/{{ cookiecutter.github_repo }})
+[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A50.30.0-brightgreen.svg)](https://www.nextflow.io/)
+[![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](http://bioconda.github.io/)
+[![Docker](https://img.shields.io/docker/automated/{{ cookiecutter.dockerhub_slug }}.svg)](https://hub.docker.com/r/{{ cookiecutter.dockerhub_slug }})
+![Singularity Container available](
+https://img.shields.io/badge/singularity-available-7E4C74.svg)
+
+### Introduction
+{{ cookiecutter.pipeline_name }}: {{ cookiecutter.pipeline_short_description }}
+
+The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker / singularity containers making installation trivial and results highly reproducible.
 
-The {{ cookiecutter.pipeline_name }} documentation is split into the following files:
 
-1. [Installation](installation.md)
+# {{ cookiecutter.pipeline_name }} Documentation
```

Comment:
It's a bit strange having the stuff above in the same readme, as it kind of duplicates material from the main `README.md`. This file is really just the index / front page for the documentation, so I think you're ok to remove the changes above this header.